app/ts/components/ui-utils.tsx

```diff
@@ -100,13 +100,17 @@ export async function openPopupOrTab(createData: browser.windows._CreateCreateDa
 	return { type: 'popup', windowOrTab: window }
 }
 
+export async function browserTabsQueryById(id: number) {
+	return (await browser.tabs.query({})).find((x) => x.id === id)
```

Comment:
`query` function could use a default here so it can just be called without the empty object